test/functional/job_configs.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, config, expected_to_fail, expected_num_subjobs, expected_num_
     teardown_build:
         - echo "Doing build teardown."
         - sleep 1
-        - ALL_SUBJOB_FILES=$(ls $PROJECT_DIR/subjob_file_*.txt)
+        - ALL_SUBJOB_FILES=$(ls $PROJECT_DIR/subjob_file_*.txt || mktemp -t clusterrunner)
```

Comment:
Had to add this when I found out that putting a slave in shutdown mode may cause it to run teardown before it has run any subjobs.